.status

```diff
@@ -4,12 +4,3 @@
 
 # Skip non-test files ending with "_test".
 packages/*: Skip
-*/packages/*: Skip
-*/*/packages/*: Skip
```

Comment:
here it might be important to keep some of these skips for the bots. The bot logic is not smart about detecting packages/* within subfolders, and we need at lest to exclude build/test/packages. Some are not needed, but it doesn't hurt to keep them